### PR TITLE
Add DBMS Learning Resource #7535

### PR DESCRIPTION
**Pull request by ys1113457623**
## What does this PR do?
Add resource(s) | Remove resource(s) | Add info | Improve repo

## Checklist:
- [ ] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [ ] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [ ] Include author(s) and platform where appropriate.
- [ ] Put lists in alphabetical order, correct spacing.
- [ ] Add needed indications (PDF, access notes, under construction).
- [ ] Used an informative name for this pull request.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

File: books/free-programming-books-subjects.md [hunk 1]
```diff
@@ -254,6 +254,15 @@ Books that cover a specific programming language can be found in the  [BY PROGRA
 * [Temporal Database Management](http://people.cs.aau.dk/~csj/Thesis/) - Christian S. Jensen
 * [The Theory of Relational Databases](http://web.cecs.pdx.edu/~maier/TheoryBook/TRD.html)
 
+### DBMS
```
Comment (eshellman) [inline books/free-programming-books-subjects.md:257]: these should go in 'database'

File: books/free-programming-books-subjects.md [hunk 2]
```diff
@@ -254,6 +254,15 @@ Books that cover a specific programming language can be found in the  [BY PROGRA
...
+
+* [Stanford Database Systems: The Complete Book](http://infolab.stanford.edu/~ullman/dscb.html) - Hector Garcia-Molina, Jeff Ullman, and Jennifer Widom(HTML)
+* [Designing Data-Intensive Applications](http://shop.oreilly.com/product/0636920032175.do) - Martin Kleppmann (HTML, PDF, EPUB, MOBI)
+* [Database Internals](https://www.oreilly.com/library/view/database-internals/9781492040330/) - Alex Petrov (PDF,HTML)
```
Comment (eshellman) [inline books/free-programming-books-subjects.md:261]: not free

File: casts/free-podcasts-screencasts-en.md [hunk 3]
```diff
@@ -353,6 +354,11 @@
 * [PG Casts](https://www.pgcasts.com) - Jack Christensen, Josh Branchaud, Jake Worth, Vidal Ekechukwu, Dorian Karter, Mary Lee, et al. Hashrocket team (screencast)
 * [Scaling Postgres](https://player.fm/series/scaling-postgres) - Ruby Tree Software Inc, Creston Jamison (podcast)
 
+### DBMS
+
+* [SQ LITE](https://www.sqlite.org/talks/wroclaw-20090310.pdf) - D. Richard Hipp, (sqlite)
```
Comment (eshellman) [inline casts/free-podcasts-screencasts-en.md:359]: not a podcast/screencast

File: casts/free-podcasts-screencasts-en.md [hunk 4]
```diff
@@ -353,6 +354,11 @@
...
+* [Data Structures and Algorithms for Big Databases](https://people.csail.mit.edu/bradley/BenderKuszmaul-tutorial-xldb12.pdf) - Michael A. Bender Bradley C. Kuszmaul MIT & Tokutek (podcast)
```
Comment (eshellman) [inline casts/free-podcasts-screencasts-en.md:360]: not a podcast/screencast

File: casts/free-podcasts-screencasts-en.md [hunk 5]
```diff
@@ -353,6 +354,11 @@
 * [PG Casts](https://www.pgcasts.com) - Jack Christensen, Josh Branchaud, Jake Worth, Vidal Ekechukwu, Dorian Karter, Mary Lee, et al. Hashrocket team (screencast)
 * [Scaling Postgres](https://player.fm/series/scaling-postgres) - Ruby Tree Software Inc, Creston Jamison (podcast)
 
+### DBMS
```
Comment (eshellman) [inline casts/free-podcasts-screencasts-en.md:357]: "Database"

**Comment by eshellman** [comment]
please have more respect for the reviewers

**Comment by ys1113457623** [comment]
I am so sry

**Pull request closed without merging**